boot/kernel_os_test.go

```diff
@@ -506,9 +507,9 @@ func (s *ubootSuite) forceUC20UbootBootloader(c *C) {
 	c.Assert(err, IsNil)
 
 	// however when finding the bootloader, since we want it to show up as the
-	// "runtime" bootloader, just use ExtractedRunKernelImage
+	// "runtime" bootloader
```

Comment:
this comment now is incomplete, but since it's obvious what the code is doing from the Role, you can just drop this comment entirely (obviously fine for a followup)

Reply:
I didn't remove it but shortened it and also clarified (hopefully) the other comment in this helper